# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-softdevice/softdevice-makefile.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- softdevice-0.5.0/Makefile~	2008-04-14 04:52:10.000000000 +0200
+++ softdevice-0.5.0/Makefile	2008-10-31 18:38:05.000000000 +0100
@@ -25,7 +25,7 @@
 VDRDIR     = ../../..
 LIBDIR     = ../../lib
 TMPDIR     = /tmp
-PLUGINLIBDIR = ./PLUGINS/lib
+PLUGINLIBDIR = /usr/lib/vdr
 
 # for older file-utils, option -f had the same effect
 CPOPTS = --remove-destination
@@ -78,7 +78,7 @@
 #   install -m 644 postprocess.h /usr/local/include/postproc/postprocess.h
 #   cp libpostproc.a /usr/local/lib/
 #
-PP_LIBAVCODEC=1
+#PP_LIBAVCODEC=1
 
 # -----------------------------------------------------------------------------
 # if you want output methods build as a single lib comment the following line
@@ -91,7 +91,7 @@
 #VIDIX_DIR    = /opt/vidix
 #VIDIX_DIR    = /usr/local/
 
-VIDIX_DIR    = /usr/local
+VIDIX_DIR    = /usr
 
 # -----------------------------------------------------------------------------
 # Set up these paths to your enviroment!
@@ -100,7 +100,7 @@
 #FFMPEGCFLAGS=-I../../../../ffmpeg/libavcodec
 #
 # If you installed ffmpeg's lib (make installlib)
-FFMPEGCFLAGS=-I/usr/local/include/ffmpeg/
+FFMPEGCFLAGS=-I/usr/include/ffmpeg/
 #FFMPEGCFLAGS=-I/home/extra/src/video/ffmpeg-0.4.8/libavcodec
 #
 #
@@ -140,19 +140,19 @@
 endif
 
 ifdef DFB_SUPPORT
-  DFB_LIBS  = -L/usr/local/lib -ldfb++
-  DFB_CFLAGS = -I/usr/local/include/dfb++ -I/usr/local/include/directfb
+  DFB_LIBS  = -L/usr/lib -ldfb++
+  DFB_CFLAGS = -I/usr/include/dfb++ -I/usr/include/directfb
   DEFINES  += -DDFB_SUPPORT
 endif
 
 ifdef CLE266_SUPPORT
-  CLE266_LIBS = -L/usr/local/lib -lcle266mpegdec
-  CLE266_CFLAGS = -I/usr/local/include
+  CLE266_LIBS = -L/usr/lib -lcle266mpegdec
+  CLE266_CFLAGS = -I/usr/include
   DEFINES  += -DHAVE_CLE266_MPEG_DECODER
 endif
 
 ifdef XV_SUPPORT
-  XV_LIBS   = -L/usr/X11R6/lib -lXi -lXext -lX11 -lm -lXv
+  XV_LIBS   = -L/usr/X11R7/lib -lXi -lXext -lX11 -lm -lXv
   ifdef LIBXDPMS_SUPPORT
     DEFINES += -DLIBXDPMS_SUPPORT
   endif
